chips/ibex-super-system/chip.toml

```diff
@@ -0,0 +1,20 @@
+[gpio0]
```

Comment:
Is there a particular datasheet that was used to populate these?

Reply:
No, I just read them out of the RTL, https://github.com/GregAC/ibex_super_system/blob/3ae1363990828a88350cd2641681dfb6d706954c/rtl/system/ibex_super_system.sv#L11.  I will add it in a comment.

Reply:
That would be helpful, thanks!